docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -399,60 +390,60 @@ sqs:
     - arn: "arn:aws:sqs:%REGION%:%ACCOUNT%:%QUEUENAME%"
       batch_size: 10
       batching_window_in_second: 0
-s3-sqs:
+s3_sqs:
```

Comment:
`s3-sqs` is the proper syntax
probably a leftover since you started the review and I pushed the latest change